app/services/claims/submit.rb

```diff
@@ -1,18 +1,25 @@
 class Claims::Submit
   include ServicePattern
 
-  def initialize(claim:, claim_params:)
+  def initialize(claim:, claim_params:, user:, link_to_claim:)
     @claim = claim
     @claim_params = claim_params
+    @user = user
+    @link_to_claim = link_to_claim
   end
 
   def call
     updated_claim.save!
+
+    unless user.type == "Claims::SupportUser"
```

Comment:
You'll still need to guard against spamming the same endpoint. This should return unless claim changed to `:submitted`